examples/DynamoDB.md

```diff
@@ -0,0 +1,44 @@
+# Using Dataloader with DynamoDB
+
+DynamoDB is a serveless key-value and document database, that you can pay only by the number of the requests you make, also it supports [batch get operations](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/DocumentClient.html#batchGet-property), making it good for use with DataLoader. But also, those batch operations have some [limitations](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BatchGetItem.html) that must be accounted for.
+
+Here is an example building a DynamoDB Dataloader using DynamoDB AWS SDK's [DocumentClient](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/DocumentClient.html). The Table used is the Movies table described at DynamoDB's official javascript [documentation](https://docs.aws.amazon.com/pt_br/amazondynamodb/latest/developerguide/GettingStarted.JavaScript.html)
+
+```js
+const AWS = require("aws-sdk");
+
+const documentClient = new AWS.DynamoDB.DocumentClient({
+    region: AWS_REGION // aws region that the dynamodb table was created e.g. us-east-1
+});
+
+const movieLoader = new Dataloader(async ids => {
```

Comment:
This will generate an error, batchGet expects a Set[] of keys, no duplicates, otherwise it will fail.